pre_commit_hooks/file_contents_sorter.py

```diff
@@ -18,19 +18,18 @@
 
 
 def sort_file_contents(f):
-    before = tuple(f)
+    before = [line.strip(b'\n\r') for line in f if line.strip()]
```

Comment:
OK sure - the use case I want this for it doesn't matter either way and having it remove blank lines/end file in newline 100% of the time makes the code simpler. Updated!